src/libraries/Microsoft.PowerFx.Connectors/Execution/FormulaValueSerializer.cs

```diff
@@ -168,37 +168,40 @@ private async Task WritePropertyAsync(string propertyName, ISwaggerSchema proper
                 {
                     foreach (DValue<RecordValue> item in tableValue.Rows)
                     {
-                        StartArrayElement(null);
-                        RecordValue rva = item.Value;
+                        if (!item.IsError)
+                        {
+                            StartArrayElement(null);
+                            RecordValue rva = item.Value;
 
-                        await WritePropertyAsync(null, propertySchema.Items, rva).ConfigureAwait(false);
+                            await WritePropertyAsync(null, propertySchema.Items, rva).ConfigureAwait(false);
+                        }
                     }
                 }
                 else if (tableValue.Rows.All(r => r.Value.Fields.Count() == 1 && r.Value.Fields.First().Name == "Value"))
                 {
                     // Working with an array of simply types
                     foreach (DValue<RecordValue> item in tableValue.Rows)
                     {
-                        StartArrayElement(null);
-                        RecordValue rva = item.Value;
-                        WriteValue(rva.Fields.First().Value);
+                        if (!item.IsError)
+                        {

```

Comment:
How does FVSerializer treat ErrorValue? Can we add a test for this?

Reply:
This code is just a safety net that was probably missing for a long time. When an ErrorValue would be present, item.Value would be null and we'd fail with a NullRef exception
For now we ignore errors and I'm not sure what sort of test we could add / what the right behavior should really be